src/ha-health-check/ha_health_check.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const http = require('http');
+const {
+  spawnSync
+} = require('child_process');
+
+http.createServer(function (req, res) {
+  const ls = spawnSync('bash', ['/var/vcap/jobs/keepalived/bin/get_keepalived_state.sh']);
```

Comment:
This is a blocking call? Having it executed every x secs is intensive. We should think of some alternatives IMO.

@jagadish-kb  @abh1kg  any thoughts?

Reply:
I guess this was done to avoid a wrong read of the state. But, in my opinion, the call from V8 can be asynchronous.

Reply:
done!